src/tbb/task_dispatcher.h

```diff
@@ -161,6 +161,9 @@ inline d1::task* task_dispatcher::steal_or_get_critical(
     if (d1::task* t = a.steal_task(arena_index, random, ed, isolation)) {
         ed.context = task_accessor::context(*t);
         ed.isolation = task_accessor::isolation(*t);
+#ifdef __TBB_STATISTICS
+        governor::get_thread_data()->add_statistics(thread_data::Statistics::STEAL_TASK);
+#endif /* __TBB_STATISTICS */
```

Comment:
I would prefer to move successful stealing statistic calculation inside the `arena_slot::steal_task` to keep all stealing-related statistics in one place (if it is possible).

Also, don't we want to calculate only the tasks that weren't respawned after finding a critical task? @pavelkumbrasev @alexey-katranov what do you think about it?

Reply:
Hello, check new version of add_statistics, please